WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -526,6 +526,20 @@ def embedable_map
     render layout: false
   end
 
+  def bookmark
+    @competition = competition_from_params
+    if !BookmarkedCompetition.exists?(competition: @competition, user: current_user)
+      BookmarkedCompetition.create(competition: @competition, user: current_user)
+    end
+    head :ok
```

Comment:
Do we do this in other places? Is this necessary because we're not rendering a page here, and just want to hit this endpoint via ajax?